.golangci.yaml

```diff
@@ -91,14 +91,17 @@ linters:
     - lll
     - misspell
     - nakedret
+    - nilerr
     - noctx
     - nolintlint
+    - prealloc
```

Comment:
@artem-streltsov actually let's disable it because nil and empty slice are different values from RPC perspective. It's ok to set capacity for slices that part of internal logic, but it might mislead and force developer to set capacity in rpc package

Reply:
Ok, should I revert changes related to this linter or leave them?